UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -3265,20 +3390,23 @@ private void OpenInTab(object obj, bool isNewTab = false, string tabTitle = null
 
             bool tabSwitched = true;
 
-            for (int i = 0; i < Tabs.Count; i++)
+            if (!isNewTab)
             {
-                if (Tabs[i].OpenedObject == obj)
+                for (int i = 0; i < Tabs.Count; i++)
                 {
-                    if (i != CurrentTabIndex)
+                    if (Tabs[i].CurrentObject == obj)
                     {
-                        CurrentTabIndex = i;
+                        if (i != CurrentTabIndex)
+                        {
+                            CurrentTabIndex = i;
 
-                        return;
-                    }
-                    else
-                        tabSwitched = false;
+                            return;
+                        }
+                        else
+                            tabSwitched = false;
 
-                    break;
+                        break;
+                    }
```

Comment:
You can invert the if check to reduce nesting

Reply:
What the if check exactly?
`if (Tabs[i].CurrentObject == obj)` ?

Reply:
For clarification, the if check in question to get inverted was `if (!isNewTab)`